### ISSUE TITLE
Incorrect content-type is displayed when uploading .js files with latest version of Azure Storage Explorer

### ISSUE THREAD

**Issue by annupurna** [microsoft/AzureStorageExplorer#2956]
**Storage Explorer Version**: 1.13.0  (this issue might be there in older versions too)
**Build Number**: 20200430.5
**Platform/OS**: Windows 10
**Architecture**: x64
**Regression From**: 1.6.2

## Bug Description ##
When uploading .js files, the content-type is displayed as 'text/plain; charset=utf-8' . 

This issue is not found in any of the versions for MAC OS. It is also not found in older versions (tested with 1.6.2 and 0.9.2) of Windows setups. 

I can see that there is a option to right click on the .js file, and change the content-type. But the requirement at my end is to upload huge help files containing hundreds  of .js files in each folder, so changing the content-type manually for each file is not an option for me. 

## Steps to Reproduce ##
Upload any  .js file to a storage blob

## Expected Experience ##
Content type should be - "application/javascript" for .js files

## Actual Experience ##
Content type is - "text/plain; charset=utf-8" for .js files

## Additional Context ##
I think the behavior in the latest version (1.13.0) should be identical in Windows and MAC OS

This is a very critical issue for our team, as we are not able to upload any help files, and our work is on-hold due to this issue. Request you to please handle this on priority.

**Comment by DanielStout5** [comment]
I'm experiencing the same issue. This breaks service workers on both Firefox and Chrome - Chrome has the most useful error message:

```
The script has an unsupported MIME type ('text/plain').
VM18 OneSignalPageSDKES6.js:1 [Service Worker Installation] Installing service worker failed SecurityError: Failed to register a ServiceWorker for scope
```

I fixed it by manually changing the content type from text/plain to application/javascript, but like annupurna said it's not realistic to manually right click each individual .js file and set its content type.

It correctly determines other file types (text/html for .html, image/x-icon for .ico, etc) so it really should also determine the content type for .js files. Ideally you could also manually edit that mapping between file extensions and content types, but even just the sane default of .js -> application/javascript would probably fix just about every use case.

**Comment by MRayermannMSFT** [comment]
Related: https://github.com/microsoft/AzureStorageExplorer/issues/2289

**Comment by MRayermannMSFT** [comment]
This has been fixed in AzCopy 10.5. We'll be updating the integrated version to that in release 1.14.1. The integration update has been merged.
